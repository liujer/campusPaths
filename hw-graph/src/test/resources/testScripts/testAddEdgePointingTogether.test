# Tests adding of edges by using two nodes
# that point to each other
CreateGraph B
AddNode B one
AddNode B two
AddEdge B one two e12
AddEdge B two one e21
ListNodes B
ListChildren B one
ListChildren B two