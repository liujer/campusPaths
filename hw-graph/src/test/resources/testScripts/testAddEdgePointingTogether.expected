# Tests adding of edges by using two nodes
# that point to each other
Created graph B
Added node one to B
Added node two to B
Added edge e12 to B
Added edge e21 to B
ListNodes of B output: one two
the children of one in B are: two(e12)
the children of two in B are: one(e21)